src/db/migrate/20170322061637_create_questions.rb

```diff
@@ -1,9 +1,10 @@
 class CreateQuestions < ActiveRecord::Migration[5.0]
   def change
     create_table :questions do |t|
-      t.text :content
+      t.string :question
```

Comment:
This is the reason why some tests and the build are failing.

Never ever change a migration file that is in the master!